app/javascript/react/components/UserShowTile.js

```diff
@@ -1,10 +1,12 @@
 import React from 'react'
 
 const UserShowTile = (props) => {
-  let user_photo = ""
-  if(props.profile_photo){
+  let user_photo = "/images/no_profile_photo.jpg"
```

Comment:
this is fine for now - in the future we could look into setting up CarrierWave to set a default image